src/portal/src/app/base/project/repository/repository-gridview.component.ts

```diff
@@ -106,6 +112,9 @@ export class RepositoryGridviewComponent implements OnChanges, OnInit, OnDestroy
               private session: SessionService,
               private router: Router,
   ) {
+    if (localStorage) {
+      this.isCardView =  localStorage.getItem(CARD_VIEW_LOCALSTORAGE_KEY) === TRUE_STR;
```

Comment:
How to use localStorage depends on whether it needs an expiration time. I recommend using sessionStorage, so that it won’t affect other users to view the repository list in the same browser.

Reply:
Session Storage may not meet the requirement(stay the same  after reload)
 "it won’t affect other users to view the repository list in the same browser"
I believe It is not common for multiple users to share one browser